tests/plugins/org.polarsys.capella.test.diagram.tools.ju/src/org/polarsys/capella/test/diagram/tools/ju/xab/CreateComponentExchangeGroup.java

```diff
@@ -77,6 +93,10 @@ public void testCreateComponentExchangeWithPorts(SessionContext context) {
   public void testCreateComponentExchangeWithoutPorts(SessionContext context) {
     testOnXAB_CreateComponentExchangeWithoutPorts(context, SA__SAB_DIAGRAM,
         BlockArchitectureExt.Type.SA, SA__SAB_A1, SA__SAB_A2);
+    

```

Comment:
can you remove the comment / just remove the test or fix it

Reply:
done